Gemfile

```diff
@@ -20,13 +20,16 @@ group :development do
 end
 
 group :test do
+  gem "govuk-content-schema-test-helpers"
   gem "simplecov", "~> 0.16"
 end
 
 group :development, :test do
   gem "byebug", "~> 10"
   gem "capybara"
+  gem "factory_bot_rails", "~> 4.0"
```

Comment:
Any particular reason for the 4.0 here? though not sure it's worth my energies trying to keep this file consistent.

Reply:
Not really - it's the latest one. Do we have a more established way to select the version?

Reply:
Just means the same as `~> 4` but it's written different to the ones pinning to those.